src/entities/session/model/store.ts

```diff
@@ -0,0 +1,21 @@
+import { createDomain, createEffect, sample } from "effector";
+import { logout, getCurrentUser } from "./api";
+import { SessionUser } from "./types";
+
+const domain = createDomain("entities/session");
+
+domain.onCreateStore((store) => {
+  store.reset(sessionClosed);
+});
+
+export const appMounted = domain.event();
+export const sessionClosed = domain.event();
+
+export const logoutFx = createEffect(logout);
```

Comment:
effects can also be created under the domain if you are using it as `domain.effect`